src/Iso8601DurationHelper/Duration.cs

```diff
@@ -678,4 +678,32 @@ private enum DurationComponent
             Seconds = Minutes + 1
         }
     }
+
+    /// <summary>
+    /// Extension class to add functionality to <see cref="DateTime"/> and <see cref="DateTimeOffset"/> so that use of <see cref="Duration"/> is similar to using <see cref="TimeSpan"/>
+    /// </summary>
+    public static class DateTimeHelper 
+    {
+        /// <summary>
+        /// Adds a <see cref="Duration"/> value to a <see cref="DateTime"/> instant
+        /// </summary>
+        /// <param name="date">The date to add to</param>
+        /// <param name="duration">The duration to add</param>
+        /// <returns></returns>
```

Comment:
Please also add a description of what it returns (maybe something similar to [`DateTime.Add(TimeSpan)`](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.add?view=net-7.0#returns))

Reply:
Done.  Very similar to DateTime[Offset].Add(TimeSpan).  Considered changing "... represented by **this** instance...", but that matches usage.  While one _could_ call an extension method directly, generally everyone uses it like the extension, right?

Reply:
Pretty much, yes. Thanks for the change!